core/builtin_constants/builtin_constants_spec.rb

```diff
@@ -61,3 +61,45 @@
     end
   end
 end
+
+describe "TRUE" do
+  ruby_version_is "3.0" do
+    it "raises NameError" do
+      -> { TRUE }.should raise_error(NameError, /uninitialized constant TRUE/)
+    end
+  end
+
+  ruby_version_is "2.7"..."3.0" do
+    it "is deprecated" do
+      -> { TRUE }.should complain(/constant ::TRUE is deprecated/)
+    end
+  end
+end
+
+describe "FALSE" do
+  ruby_version_is "3.0" do
+    it "raises NameError" do
+      -> { FALSE }.should raise_error(NameError, /uninitialized constant FALSE/)
+    end
+  end
+
+  ruby_version_is "2.7"..."3.0" do
+    it "is deprecated" do
+      -> { FALSE }.should complain(/constant ::FALSE is deprecated/)
+    end
+  end
+end
+
+describe "NIL" do
+  ruby_version_is "3.0" do
+    it "raises NameError" do
+      -> { NIL }.should raise_error(NameError, /uninitialized constant NIL/)
+    end
+  end
+
+  ruby_version_is "2.7"..."3.0" do
+    it "is deprecated" do
+      -> { NIL }.should complain(/constant ::NIL is deprecated/)
+    end
+  end
+end
```

Comment:
It should be in `language/predefined_spec.rb`, could you move them there?
(There were specs for it incorrectly removed in b3ac8c8ab5c51644562812e9364d36acc614132)